news/2019-08-01-cwc-2019-preview-grand-final.md

```diff
@@ -0,0 +1,43 @@
+
+The second Loser's Bracket SemiFinal pitted a resurgent and highly-rated France against perhaps the greatest success story of this year's tournament in Mexico. France flexed their muscles on converted osu! difficulties throughout, but struggled to win key games in the Hidden and DoubleTime pools to claim victory. Interestingly for two teams both highly rated on the HardRock mod, not a single pick from the mod bracket was selected by either team. Mexico claimed a 7-2 victory and a spot in the final four, having never previously advanced beyond the Quarter Final stage.
+
+This left Chile and Mexico - two teams that quite convincingly beat more favoured opposition - to battle it out for a spot in the Grand Finals. Both teams traded blows throughout, and despite several break points neither was able to take a commanding lead over the other. Chile once again exhausted the Hidden Pool early, with Mexico opting for HardRock and DoubleTime to outpace their opponents. Mexican dreams were very much alive when they secured a 5th game on the ever-dramatic Deconstruction Star by a margin of 3,752 points, but it was ultimately in the NoMod pool where the match was decided, as the Chilean roster's superior consistency shone through.
+
+![](/wiki/shared/news/2019-08-01-cwc-2019-preview-grand-final/CLvMX-game10-720w.jpg)
+
+An eventual 7-5 victory took them to their second consecutive Loser's Bracket Final next week, once against against China - can they go one (or even two!) better this year? Frankly, if you watch no other match from this year's osu!catch World Cup thus far, make this one a priority - it really was _that_ spectacular.
+
+## Summary
+
+Only two matches remain before this year's osu!catch World Cup winners are decided, and you can catch both of them live in all their glory on the [osu!live Twitch channel](https://www.twitch.tv/osulive) this coming weekend. The Losers' Bracket Final between China and Chile will take place at **16:00 UTC+0 this Saturday, August 3rd**.
+
+The Grand Final will be at **01:00 UTC+0 on Sunday, August 4th** if Chile are victorious in the Losers' Bracket Final, and **12:00 UTC+0 on Sunday** if China come out on top. Be sure to follow [@osugame on Twitter](https://twitter.com/osugame) to receive notifications of all updates, scheduling and live broadcasts!
+
+If you haven't yet picked up a wicked [profile banner](https://osu.ppy.sh/store/products/608) to show your undying love and support for your favourite teams and players from this year's osu!catch World Cup yet, hurry over to the [osu!store](https://osu.ppy.sh/store/listing) now, as they will shortly become unavailable! All proceeds go towards helping make osu! and World Cups in particular even more awesome, so don't delay!
+
+I hope to see you all cheering our finalists on this weekend for the fantastic conclusion to another year of incredible fruit catching!
```

Comment:
```suggestion
I hope to see you all cheering our finalists this weekend for the fantastic conclusion to another year of incredible fruit catching!
```

Reply:
was originally going to say "on Saturday" but then changed my mind, fixed